shell/imports/sandstorm-db/db.js

```diff
@@ -24,6 +24,7 @@ import { Random } from "meteor/random";
 import { SHA256 } from "meteor/sha";
 import { HTTP } from "meteor/http";
 import { iconSrcForPackage } from "/imports/sandstorm-identicons/helpers.js";
+import { globalDb } from "/imports/db-deprecated.js";
```

Comment:
Ummm, I think I intended this import to go in the profile.js file, where `globalDb` is used.